test/hotspot/jtreg/TEST.groups

```diff
@@ -138,15 +156,14 @@ serviceability_ttf_virtual = \
   -serviceability/jvmti/events  \
   -serviceability/jvmti/negative
 
+# gtest and non_hotspot tests shouldn't be executed with VM flags
+# hotspot_misc includes sanity testing and might be used
+# for execution non-component tests with VM flags
 tier1_common = \
-  sources \
-  sanity/BasicVMTest.java \
-  gtest/GTestWrapper.java \
-  gtest/LockStackGtests.java \
-  gtest/MetaspaceGtests.java \
-  gtest/LargePageGtests.java \
-  gtest/NMTGtests.java \
-  gtest/WindowsProcessorGroups.java
+  sanity \
+  native_sanity \
+  :non_hotspot_tests \
```

Comment:
Why should we want to execute the non-hotspot-tests in tier1 ??

Reply:
That's a good question. These tests are executed now in tier1. They include source verification and test library tests. We could introduce tier2_common for them, do you want to do it in this RFE?

Reply:
It wasn't apparent to me that we already run the testlibrary_tests in tier1. That makes little sense to me - it is akin to running jtreg tests. The only time we need to run the testlibrary_tests is when we make changes to the test library, so a higher level tier would make more sense to me. That said, the intent of this PR is not, IIUC, to make that kind of change so that is perhaps better left to a separate change.

The sources tests should be run in tier1.